public/data/varghesejose2020.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "Varghese Jose",
+  "type": "personal",
+  "bio": "Open Source Contributor||Full Stack Developer||FOSS Enthusiast",
```

Comment:
This is a small suggestion, because I think it'll look weird on the site
```suggestion
  "bio": "Open Source Contributor | Full Stack Developer | FOSS Enthusiast",
```

Reply:
:+1: